webapps/frontend/app/viewmodels/wizards/addpreset/gather.js

```diff
@@ -24,10 +24,14 @@ define([
         self.data = data;
 
         // Computed
+        self.data.canEdit = ko.computed(function() {
```

Comment:
You're overwriting the normal observable defined in ```data.js``` here. You should just place the computed in ```data.js``` from the beginning